db/migrate/20220730140822_create_account_objectives.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAccountObjectives < ActiveRecord::Migration[6.0]
+  def change
+    create_table :account_objectives do |t|
+      t.string :target_name, null: false
```

Comment:
I think we don't need `target_name` because we can get this data from `accounts.name`:

```suggestion
```

![image](https://user-images.githubusercontent.com/14297526/188151783-98693095-7bc4-4bc9-a5af-808fa8e9abf4.png)